plugins/kobolight.koplugin/main.lua

```diff
@@ -148,4 +159,39 @@ function KoboLight:onSwipe(_, ges)
     return true
 end
 
+function KoboLight:addToMainMenu(tab_item_table)
+    table.insert(tab_item_table.plugins, {
+        text = _("Frontlight gesture controller"),
+        callback = function()
+            local image = ImageWidget:new{
+                file = PluginLoader.plugin_path .. "/kobolight.koplugin/demo.png",
+                height = Screen:getHeight(),
+                width = Screen:getWidth(),
+                scale_factor = 0,
+            }
+            UIManager:show(image)
+            UIManager:show(ConfirmBox:new{
+                text = T(_("Frontlight gesture controller can:\n- Turn on or off frontlight by tapping left-bottom of the screen.\n- Change frontlight intensity by swiping up or down on the left of the screen.\n\nDo you want to %1 it?"),
```

Comment:
"bottom left" sounds more natural, to me, don't know if there is an international/industrial convention for such terms.   Auto translation may need some oversight.